identities/views.py

```diff
@@ -257,6 +258,23 @@ def perform_create(self, serializer):
         return serializer.save(created_by=self.request.user)
 
 
+class OptOutSearchList(generics.ListAPIView):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = OptOutSerializer
+
+    def get_queryset(self):
+        query_params = list(self.request.query_params.keys())
+        filter_criteria = {}
+
+        for filter in query_params:
+            filter_criteria[filter] = self.request.query_params[filter]
```

Comment:
Can we not use the `.dict()` method? https://docs.djangoproject.com/en/1.10/ref/request-response/#django.http.QueryDict.dict